muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/workspaces/index.ts

```diff
@@ -450,94 +517,122 @@ export interface LoadMoreCurrentWorkspaceJournalsFromServerTriggerType {
 }
 
 export interface LoadWholeWorkspaceMaterialsTriggerType {
-  (workspaceId: number, includeHidden: boolean, callback?:(nodes: Array<MaterialContentNodeType>)=>any):AnyActionType
+  (workspaceId: number, includeHidden: boolean, callback?: (nodes: Array<MaterialContentNodeType>) => any): AnyActionType
 }
 export interface LoadWholeWorkspaceHelpTriggerType {
-  (workspaceId: number, includeHidden: boolean, callback?:(nodes: Array<MaterialContentNodeType>)=>any):AnyActionType
+  (workspaceId: number, includeHidden: boolean, callback?: (nodes: Array<MaterialContentNodeType>) => any): AnyActionType
 }
 export interface SetWholeWorkspaceMaterialsTriggerType {
   (materials: MaterialContentNodeListType): AnyActionType
 }
 export interface SignupIntoWorkspaceTriggerType {
   (data: {
-    success: ()=>any,
-    fail: ()=>any,
+    success: () => any,
+    fail: () => any,
     workspace: WorkspaceType,
     message: string,
-  }):AnyActionType
+  }): AnyActionType
 }
 export interface SetCurrentWorkspaceMaterialsActiveNodeIdTriggerType {
-  (id: number):AnyActionType
+  (id: number): AnyActionType
 }
 export interface LoadWorkspaceCompositeMaterialReplies {
-  (id: number):AnyActionType
+  (id: number): AnyActionType
 }
 export interface UpdateAssignmentStateTriggerType {
-  (successState: MaterialCompositeRepliesStateType, avoidServerCall: boolean, workspaceId: number, workspaceMaterialId: number, existantReplyId?: number, successMessage?: string, callback?: ()=>any):AnyActionType
+  (successState: MaterialCompositeRepliesStateType, avoidServerCall: boolean, workspaceId: number, workspaceMaterialId: number, existantReplyId?: number, successMessage?: string, callback?: () => any): AnyActionType
 }
 
 export interface LoadUserWorkspaceEducationFiltersFromServerTriggerType {
-  (loadOrganizationWorkspaces:boolean):AnyActionType
+  (loadOrganizationWorkspaces: boolean): AnyActionType
+}
+
+export interface setFiltersTriggerType {
+  (loadOrganizationWorkspaceFilters: boolean, filters: WorkspaceStateFilterListType): AnyActionType
 }
 
 export interface LoadUserWorkspaceCurriculumFiltersFromServerTriggerType {
-  (loadOrganizationWorkspaceFilters:boolean, callback?: (curriculums: WorkspaceCurriculumFilterListType)=>any):AnyActionType
+  (loadOrganizationWorkspaceFilters: boolean, callback?: (curriculums: WorkspaceCurriculumFilterListType) => any): AnyActionType
 }
 
+
+
 export interface LoadUserWorkspaceOrganizationFiltersFromServerTriggerType {
-  (callback?: (organizations: WorkspaceOrganizationFilterListType) => any):AnyActionType;
+  (callback?: (organizations: WorkspaceOrganizationFilterListType) => any): AnyActionType;
 }
 
 export interface UpdateWorkspaceTriggerType {
   (data: {
     workspace: WorkspaceType,
     update: WorkspaceUpdateType,
-    success?: ()=>any,
-    fail?: ()=>any
-  }):AnyActionType
+    activeFilters?: WorkspacesActiveFiltersType,
+    addStudents?: SelectItem[],
```

Comment:
SelectItem is a UI specific element and should not be used in the action.

Reply:
It's a general type for a selectable item. Not a user, not a workspace, not a message, but any of them. Suggestions?

Reply:
It does not exist anymore, but github does not care